tests/test_primitive.py

```diff
@@ -4,6 +4,7 @@
 import equinox.internal as eqxi
 import jax
 import jax.core
```

Comment:
Thank you for the fix!

By the way, do we need to keep all of the `import jax.core` around? I'm guessing most of these can be removed.

Reply:
Done

Reply:
Seems you failed to push after removing the unused imports?

Reply:
@garymm No, a fair amount of files still need jax.core.